projects/sdk-core/src/constants/abi/ERC20.json

```diff
@@ -0,0 +1,117 @@
+[
```

Comment:
Should these and other src/constants be committed?

Reply:
yes, we can commit the abis, these are pretty static